src/spikeinterface/sorters/external/kilosort4.py

```diff
@@ -227,39 +227,14 @@ def _run_from_folder(cls, sorter_output_folder, params, verbose):
         torch.cuda.manual_seed_all(1)
         torch.random.manual_seed(1)
         # if not params["skip_kilosort_preprocessing"]:
-        if params["do_correction"]:
-            # this function applies both preprocessing and drift correction
-            ops, bfile, st0 = compute_drift_correction(
-                ops, device, tic0=tic0, progress_bar=progress_bar, file_object=file_object
-            )
-        else:
+        if not params["do_correction"]:
             print("Skipping drift correction.")
-            hp_filter = ops["preprocessing"]["hp_filter"]
-            whiten_mat = ops["preprocessing"]["whiten_mat"]
-
-            bfile = BinaryFiltered(
-                ops["filename"],
-                n_chan_bin,
-                fs,
-                NT,
-                nt,
-                twav_min,
-                chan_map,
-                hp_filter=hp_filter,
-                whiten_mat=whiten_mat,
-                device=device,
-                do_CAR=do_CAR,
-                invert_sign=invert,
-                dtype=dtype,
-                tmin=tmin,
-                tmax=tmax,
-                artifact_threshold=artifact,
-                file_object=file_object,
-            )
+            ops["nblocks"] = 0
```

Comment:
This is the KS based solution for skipping drift correction right? 

I think it does make things simpler to just rely on their machinery. Maybe this will speed things up too from the wrapper?

Reply:
yes, let's see :)